src/woff2_dec.cc

```diff
@@ -24,6 +24,7 @@
 #include <string>
 #include <vector>
 #include <map>
+#include <unordered_set>
```

Comment:
@ rsheeter What was the rationale for adding this header? I don't see any use of unordered_set in this file...